pombola/south_africa/templates/south_africa/local_rep_list_item.html

```diff
@@ -0,0 +1,84 @@
+{% load staticfiles %}
+{% load thumbnail %}
+
+<li id="{{ person_data.element_id }}">
+    {# Display a photo if one is available... #}
+    {% if person_data.person %}
+      <a href="{{ person_data.person.get_absolute_url }}">
+      {% thumbnail person_data.person.primary_image "58x58" crop="center" as im %}
+        <img class="constituency-office-mp-photo" src="{{ im.url }}" width="{{ im.width }}" height="{{ im.height }}"/>
+      {% empty %}
+        <img class="constituency-office-mp-photo" src="{% static 'images/person-90x90.jpg' %}" width="58" height="58" />
+      {% endthumbnail %}
+    {% else %}
+        <img class="constituency-office-mp-photo" src="{% static 'images/person-90x90.jpg' %}" width="58" height="58" />
```

Comment:
What does it mean if there is no `person_data.person`? Should we still use a blank person image, or something else?